ui-tests/src/test/java/io/syndesis/qe/steps/other/GMailSteps.java

```diff
@@ -0,0 +1,44 @@
+package io.syndesis.qe.steps.other;
+
+import com.google.api.services.gmail.model.Message;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Then;
+import io.syndesis.qe.utils.GMailUtils;
+import lombok.extern.slf4j.Slf4j;
+import org.assertj.core.api.Assertions;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+import java.io.IOException;
+import java.util.List;
+
+@Slf4j
+public class GMailSteps {
+
+    @Then("^.*send an e-mail$")
+    public void sendMail() {
+        GMailUtils.sendEmail("me", "jbossqa.fuse@gmail.com", "syndesis-tests", "Red Hat");
+    }
+
+    @And("^delete emails from \"([^\"]*)\" with subject \"([^\"]*)\"$")
```

Comment:
"And" means nothing, it is just a "shortcut" that defines that this step is the same type as previous one

Reply:
this "@And" should still be changed to something else :-)